### PR TITLE
Fix husky

### PR DESCRIPTION
**Pull request by Javitocatral**

### CODE REVIEW

File: .vscode/settings.json
Comment (zwergius) [inline .vscode/settings.json:1]: gitignore

File: src/lib/components/Hoverable.svelte [hunk 1]
```diff
@@ -10,6 +10,6 @@
   }
 </script>
 
-<div on:mouseenter="{enter}" on:mouseleave="{leave}">
+<div on:mouseenter={enter} on:mouseleave={leave} role="button" tabindex="0">
```
Comment (zwergius) [inline src/lib/components/Hoverable.svelte:13]: not sure about this

File: src/lib/components/ClientsSticker.svelte [hunk 2]
```diff
@@ -86,13 +86,15 @@
 {#if windowHeight}
   <div
     use:pannable
-    on:panmove={handlePanMove}
-    on:panstart={handlePanStart}
-    on:panend={handlePanEnd}
     on:mouseenter={handleMouseEnter}
+    role="button"
```
Comment (zwergius) [inline src/lib/components/ClientsSticker.svelte:90]: this is not a button

File: src/lib/components/ThemeSelector.svelte [hunk 3]
```diff
@@ -36,7 +36,7 @@
   class="button"
   on:click={() => setColorMode('dark')}
   on:keypress={(e) => handleKeypress(e, 'dark')}
-  tabIndex="0"
+  tabindex="0"
```
Comment (zwergius) [inline src/lib/components/ThemeSelector.svelte:39]: we should probably change these divs tu button tags and avoid this junk

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/Hoverable.svelte (hunk 1 above)
Comment (Javitocatral) [reply src/lib/components/Hoverable.svelte:13]: mouseleave handlers must have an ARIA role(a11y-no-static-element-interactions)
Elements with the 'button' interactive role must have a tabindex value.(a11y-interactive-supports-focus)
These are the errors, one for not having a role and another for the tabindex. It seems like it's now mandatory, at least with the button role.

**Review by zwergius — APPROVED** [review]

**Pull request merged**